x-pack/platform/packages/shared/kbn-elastic-assistant/impl/assistant/api/knowledge_base/use_knowledge_base_status.tsx

```diff
@@ -38,7 +39,10 @@ export const useKnowledgeBaseStatus = ({
   resource,
   toasts,
   enabled,
-}: UseKnowledgeBaseStatusParams): UseQueryResult<ReadKnowledgeBaseResponse, IHttpFetchError> => {
+}: UseKnowledgeBaseStatusParams): UseQueryResult<
+  ReadKnowledgeBaseResponse & { product_documentation_status: InstallationStatus },
```

Comment:
Think product_documentation_status would come from here already https://github.com/elastic/kibana/pull/212463/files#diff-360d03de6632fda33b31cfd0093447174be39e2dea0e425599dbf0e3e766f8baR76
```suggestion
  ReadKnowledgeBaseResponse,
```

Reply:
it returns `string` instead of actual values, because they are defined outside of this package